### PR TITLE
Fix android

### PR DESCRIPTION
**Pull request by pthom**
With a few adpatations I could run your project under android!

I have tested it with arm and arm64, with the window and widget demos and ut runs perfectly.

Your project is seriously cool, and it is refreshing to be able to create a c++ app with 10 lines of code on a tablet.

cheers

### CODE REVIEW

**Comment by seanchas116** [comment]
Thank you for the cool pull request!
I don't have Android devices so I tried to run it in Android simulator but I got runtime errors:

```
E eglCodecCommon: glUtilsParamSize: unknow param 0x000085b5
```

I don't know why this error is happening. Do you know something about this?
I think it's ok to support physical devices only but it would be nice if it runs on simulators as well.

**Comment by pthom** [comment]
Hello,

Thank you for your answer! 

I cannot currently test on an emulator (my qt creator is installed on a virtual machine, and I cannot install an emulator on a virtual machine).

I suspect this is a limitation of OpenGL in an emulator: 

May be you could uncheck the setting "Use host GPU".
See  
https://stackoverflow.com/questions/22348801/phonegap-eclipse-issue-eglcodeccommon-glutilsparamsize-unknow-param-errors 
and
https://stackoverflow.com/questions/51767154/android-gradle-build-faild-with-this-error 


https://stackoverflow.com/questions/39057187/android-app-error-glutilsparamsize-unknow-param

File: ImGuiRenderer.cpp [hunk 1]
```diff
@@ -212,7 +223,7 @@ bool ImGuiRenderer::createDeviceObjects()
         "}\n";
 
     const GLchar* fragment_shader =
-        "#version 330\n"
+        IMGUIRENDERER_GLSL_VERSION
```
Comment (seanchas116) [inline ImGuiRenderer.cpp:226]: I finally found the reason why no window is show in emulator.
`precision mediump float;` is needed in fragment shader.
(Seems `E eglCodecCommon: glUtilsParamSize: unknow param 0x000085b5` was not related)

Comment (pthom) [reply ImGuiRenderer.cpp:226]: I added a line precision mediump float; in the fragment shader. Tell me if it is ok for you

**Comment by seanchas116** [comment]
Thank you! It's working 👍 

image

**Pull request merged**